smpe/bld/service/current-hold.txt

```diff
@@ -0,0 +1,20 @@
+****************************************************************
```

Comment:
The file name should be current-hold-ACTION.txt so that the build knows how to classify this.